### ISSUE TITLE
In the transcription toolbar change the field help hover to be triggered on the label instead of 'i' button

### ISSUE THREAD

**Issue by mcagan** [open-data-rescue/climate-data-rescue#241]
**Describe the bug**
In the transcription toolbar, the field help hover should be triggered anywhere on the label instead of on the info button.

**Screenshots**
<img width="1189" alt="Screen Shot 2020-06-18 at 6 18 06 PM" src="https://user-images.githubusercontent.com/43585486/85077527-826fe680-b190-11ea-8b41-52912410ef7d.png">
<img width="1159" alt="Screen Shot 2020-06-18 at 6 18 12 PM" src="https://user-images.githubusercontent.com/43585486/85077537-8b60b800-b190-11ea-9857-d832ee48556d.png">

**Comment by rsmithlal** [comment]
These are the docs for the bootstrap 3 popovers https://getbootstrap.com/docs/3.3/javascript/#popovers

**Comment by rsmithlal** [comment]
This is the info icon tag for the date/time https://github.com/open-data-rescue/climate-data-rescue/blob/41b98a0c338dc405b8856b330e7948f4bd65bbce/app/views/shared/_transcriber_templates.html.erb#L194

**Comment by rsmithlal** [comment]
This is the info icon tag for the field labels https://github.com/open-data-rescue/climate-data-rescue/blob/41b98a0c338dc405b8856b330e7948f4bd65bbce/app/views/shared/_transcriber_templates.html.erb#L206

**Comment by rsmithlal** [comment]
To complete the work for this issue, you will want to move the attributes from [line 208](https://github.com/open-data-rescue/climate-data-rescue/blob/41b98a0c338dc405b8856b330e7948f4bd65bbce/app/views/shared/_transcriber_templates.html.erb#L208) to [line 215](https://github.com/open-data-rescue/climate-data-rescue/blob/41b98a0c338dc405b8856b330e7948f4bd65bbce/app/views/shared/_transcriber_templates.html.erb#L215) from the `i` tag to either the `.scribe_input_label` tag on [line 205](https://github.com/open-data-rescue/climate-data-rescue/blob/41b98a0c338dc405b8856b330e7948f4bd65bbce/app/views/shared/_transcriber_templates.html.erb#L205) (using the one line per attribute whitespace) or wrap the actual field name [line 216](https://github.com/open-data-rescue/climate-data-rescue/blob/41b98a0c338dc405b8856b330e7948f4bd65bbce/app/views/shared/_transcriber_templates.html.erb#L216) in a span tag and move the mentioned attributes to that tag.

The difference between adding them to the div and to the span are that when on the div the popover will trigger on mouse over on the whole div including white space, and on the span it will only appear when the text itself is hovered.

Give both a try and see the difference!

To summarize the popovers, they use the title attribute for the popover title and the data-content attribute for the main popover body text. the other data-attributes control other behaviour for the plugin such as what direction it pops from or what the trigger is (click or hover).

If you have any questions, please comment here so that there is a public record of it for the future =)

**Comment by rsmithlal** [comment]
Please remove the `i` tags that have the icon class once you get the popover working on the label. Also please ensure to make the change for both the separate date/time info bubble and the field labels that are dynamically added to the form using Rails templates.

**Comment by mcagan** [comment]
I implemented the fix using the label tag as you suggested. It works well for the field labels but for the date/time label the help text is showing HTML tags as text (I'm attaching a picture below). I'm trying to figure out how to fix that but if you have any advice it would help me a lot :) 
<img width="1015" alt="Screen Shot 2020-06-19 at 1 49 27 PM" src="https://user-images.githubusercontent.com/43585486/85165369-b8b27200-b233-11ea-921b-1c065400b3ee.png">

**Comment by mcagan** [comment]
Partially fixed the issue, now it's not showing the text in the <nobr> tag. Will keep this thread updated.
<img width="1072" alt="Screen Shot 2020-06-19 at 2 11 11 PM" src="https://user-images.githubusercontent.com/43585486/85167458-dc2aec00-b236-11ea-9779-9867b7f9aec6.png">
